docs/source/user-guide/configuration/configuration-settings.rst

```diff
@@ -0,0 +1,823 @@
+
+.. _config-channels:
+
+Channel locations (channels)
+----------------------------
+
+Listing channel locations in the ``.condarc`` file overrides
+conda defaults, causing conda to search only the channels listed
+here, in the order given.
+
+Use ``defaults`` to automatically include all default channels.
+Non-URL channels are interpreted as Anaconda.org user or organization
+names. You can change this by modifying the ``channel_alias`` as described
+in :ref:`set-ch-alias`. The default is just ``defaults``.
+
+**Example:**
+
+.. code-block:: yaml
+
+  channels:
+    - <anaconda_dot_org_username>
+    - http://some.custom/channel
+    - file:///some/local/directory
+    - defaults
+
+To select channels for a single environment, put a ``.condarc``
+file in the root directory of that environment (or use the
+``--env`` option when using ``conda config``).
+
+**Example:** If you have installed Miniconda with Python 3 in your
+home directory and the environment is named "flowers", the
+path may be::
+
+  ~/miniconda3/envs/flowers/.condarc
+
+.. _default-channels:
+
+Default channels (default_channels)
+-----------------------------------
+
+Normally the defaults channel points to several channels at the
+`repo.anaconda.com <https://repo.anaconda.com/>`_ repository, but if
+``default_channels`` is defined, it sets the new list of default channels.
+This is especially useful for airgapped and enterprise installations:
+
+To ensure that all users only pull packages from an on-premises
+repository, an administrator can set both :ref:`channel alias <channel-alias>` and
+``default_channels``.
+
+.. code-block:: yaml
+
+  default_channels:
+    - http://some.custom/channel
+    - file:///some/local/directory
+
+.. _auto-update-conda:
+
+Update conda automatically (auto_update_conda)
+----------------------------------------------
+
+When ``True``, conda updates itself any time a user updates or
+installs a package in the root environment. When ``False``,
+conda updates itself only if the user manually issues a
+``conda update`` command. The default is ``True``.
+
+**Example:**
+
+.. code-block:: yaml
+
+  auto_update_conda: False
+
+.. _always-yes:
+
+Always yes (always_yes)
+-----------------------
+
+Choose the ``yes`` option whenever asked to proceed, such as
+when installing. Same as using the ``--yes`` flag at the
+command line. The default is ``False``.
+
+**Example:**
+
+.. code-block:: yaml
+
+  always_yes: True
+
+.. _show-channel-urls:
+
+Show channel URLs (show_channel_urls)
+-------------------------------------
+
+Show channel URLs when displaying what is going to be downloaded
+and in ``conda list``. The default is ``False``.
+
+**Example:**
+
+.. code-block:: yaml
+
+  show_channel_urls: True
+
+.. _change-command-prompt:
+
+Change command prompt (changeps1)
+---------------------------------
+
+When using ``conda activate``, change the command prompt from ``$PS1``
+to include the activated environment. The default is ``True``.
```

Comment:
```suggestion
When using ``conda activate``, it changes the command prompt from its default
to include the name of the activated environment. This is on by default. Set this to ``False``
to disable.
```

Reply:
@kathatherine, is that better?

Reply:
It's not totally clear what "it" is here. I assume the command `conda activate`. Any objections to:

```suggestion
``conda activate`` changes the command prompt from its default 
the name of the activated environment. This is on by default. Set this to ``False`` 
to disable.
```
Might be misunderstanding, but this setting, when True, essentially replaces (base) with whatever environment name is activated, so I chose to remove "to include". Feel free to ignore this last part and put "to include" back in if I've missed something. Include feels like an appending to me, rather than a full replace, if that makes sense?

Reply:
I think I'm going to leave this as-is for now sake of not complicating the pull request more. I believe this entire page will be a good candidate to come back and review later along with the "Managing Environments" page.